src/openrct2/platform/linux.c

```diff
@@ -56,6 +56,8 @@ void platform_get_exe_path(utf8 *outPath, size_t outSize)
         log_fatal("failed to get process path");
 
     }
+#elif defined(__OpenBSD__)
+    strlcpy(exePath, "/usr/local/bin/", sizeof(exePath));
```

Comment:
Yes, can do something like get dirname of executable and check against it. Should it just warn, or should it error?